getting_started/dev-environment/ubuntu.md

```diff
@@ -0,0 +1,42 @@
+# Instructions for Ubuntu
+
+Run the following to install most of the needed packages:
+
+```shell
+$ sudo apt-get install python python-setuptools python-dev build-essential \
+                       python-software-properties python-pip vagrant virtualbox
+```
+
+Add the official nodejs ppa, then install nodejs and its package manager, npm:
+
+```shell
+$ curl -sL https://deb.nodesource.com/setup_7.x | sudo -E bash -
+$ sudo apt-get install nodejs
+```
+
+Install virtualenv and virtualenvwrapper:
+
+```shell
+$ pip install virtualenv virtualenvwrapper
+```
+
+Add the following to your `~/.bashrc`:
+
+```shell
+export WORKON_HOME=~/.virtualenvs
+. /usr/local/bin/virtualenvwrapper.sh
+```
+
+Activate virtualenvwrapper:
+
+```shell
+$ source ~/.bashrc`
+```
+
+Install ansible:
+
+```shell
+$ pip install ansible
```

Comment:
Will add. Forgot that we aren't currently in a virtual environment yet (which does not require sudo).